Sprint-1/destructuring/exercise-1/exercise.js

```diff
@@ -3,7 +3,7 @@ const personOne = {
   age: 34,
   favouriteFood: "Spinach",
 };
-
+let {name, age, favouriteFood} = personOne;
```

Comment:
Nice one with this. Just a little tip with const and let: use const by default and only use let if you need to re-assign a value.

https://stackoverflow.com/questions/41086633/in-javascript-why-should-i-usually-prefer-const-to-let#:~:text=Basically%2C,matter%20of%20style